lib/dynflow/execution_history.rb

```diff
@@ -12,7 +12,11 @@ class ExecutionHistory
 
     module Event
       def inspect
-        "#{Time.at(time).utc}: #{name}".tap { |s| s << " @ #{world_id}" if world_id }
+        if world_id
+          "#{Time.at(time).utc}: #{name} @ #{world_id}"
+        else
+          "#{Time.at(time).utc}: #{name}"
+        end
```

Comment:
Could we do something like this to reduce repetition?
```suggestion
        [ "#{Time.at(time).utc}: #{name}", world_id ].compact.join(' @ ')
```

Reply:
I prefer faster code, but this is indeed more of a debugging part so why not.